src/cpp/session/include/session/SessionSourceDatabase.hpp

```diff
@@ -200,7 +200,8 @@ struct Events : boost::noncopyable
    boost::signal<void(const std::string&,
                       boost::shared_ptr<SourceDocument>)>      onDocRenamed;
    boost::signal<void(const std::string&)>                     onDocAdded;
-   boost::signal<void(const std::string&)>                     onDocPendingRemove;
+   boost::signal<void(const std::string&,
```

Comment:
This parameter is unnecessary now (since it's accessible from `SourceDocument::id()`)